### ISSUE TITLE
Documentation access with ? or ?? not available

### ISSUE THREAD

**Issue by videlec** [conda-forge/sage-feedstock#40]
Issue: in the Sage console one should be able to access the documentation with a single question mark `?` and the source code with two `??`. However the following gives an error
```
sage: a = 1
sage: a?
ApplicationError                          Traceback (most recent call last)
<ipython-input-8-1cb05c887abe> in <module>()
----> 1 get_ipython().magic('pinfo2 a')
/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/interactiveshell.py in magic(self, arg_s)
   2158         magic_name, _, magic_arg_s = arg_s.partition(' ')
   2159         magic_name = magic_name.lstrip(prefilter.ESC_MAGIC)
-> 2160         return self.run_line_magic(magic_name, magic_arg_s)
   2161 
   2162     #-------------------------------------------------------------------------

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/interactiveshell.py in run_line_magic(self, magic_name, line)
   2079                 kwargs['local_ns'] = sys._getframe(stack_depth).f_locals
   2080             with self.builtin_trap:
-> 2081                 result = fn(*args,**kwargs)
   2082             return result
   2083 

</home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/decorator.py:decorator-gen-75> in pinfo2(self, parameter_s, namespaces)

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/magic.py in <lambda>(f, *a, **k)
    186     # but it's overkill for just that one bit of state.
    187     def magic_deco(arg):
--> 188         call = lambda f, *a, **k: f(*a, **k)
    189 
    190         if callable(arg):

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/magics/namespace.py in pinfo2(self, parameter_s, namespaces)
     67         '%pinfo2 object' is just a synonym for object?? or ??object."""
     68         self.shell._inspect('pinfo', parameter_s, detail_level=1,
---> 69                             namespaces=namespaces)
     70 
     71     @skip_doctest

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/interactiveshell.py in _inspect(self, meth, oname, namespaces, **kw)
   1534             elif meth == 'pinfo':
   1535                 pmethod(info.obj, oname, formatter, info, 
-> 1536                         enable_html_pager=self.enable_html_pager, **kw)
   1537             else:
   1538                 pmethod(info.obj, oname)

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/oinspect.py in pinfo(self, obj, oname, formatter, info, detail_level, enable_html_pager)
    713         - detail_level: if set to 1, more information is given.
    714         """
--> 715         info = self._get_info(obj, oname, formatter, info, detail_level)
    716         if not enable_html_pager:
    717             del info['text/html']

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/oinspect.py in _get_info(self, obj, oname, formatter, info, detail_level)
    614         """Retrieve an info dict and format it."""
    615 
--> 616         info = self._info(obj, oname=oname, info=info, detail_level=detail_level)
    617 
    618         _mime = {
/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/IPython/core/oinspect.py in _info(self, obj, oname, info, detail_level)
    763                     ds += "\nDocstring:\n" + obj.__doc__
    764         else:
--> 765             ds = getdoc(obj)
    766             if ds is None:
    767                 ds = '<no docstring>'

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sage/misc/lazy_import.pyx in sage.misc.lazy_import.LazyImport.__call__ (build/cythonized/sage/misc/lazy_import.c:3696)()
    352             True
    353         """
--> 354         return self.get_object()(*args, **kwds)
    355 
    356     def __repr__(self):

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sage/misc/sageinspect.py in sage_getdoc(obj, obj_name, embedded_override)
   1930         return ''
   1931     r = sage_getdoc_original(obj)
-> 1932     s = sage.misc.sagedoc.format(r, embedded=(embedded_override or EMBEDDED_MODE))
   1933 
   1934     # Fix object naming

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sage/misc/sagedoc.py in format(s, embedded)
    709             s = process_mathtt(s)
    710         s = process_extlinks(s, embedded=embedded)
--> 711         s = detex(s, embedded=embedded)
    712     return s
    713 

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sage/misc/sagedoc.py in detex(s, embedded)
    226         s = _rmcmd(s, 'mathop')
    227         s = _rmcmd(s, 'mathrm')
--> 228         s = sphinxify(s, format='text')
    229         # Do math substitutions. The strings to be replaced should be
    230         # TeX commands like "\\blah". Do a regular expression

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sage/misc/sphinxify.py in sphinxify(docstring, format)
     97     # buildername, confoverrides, status, warning, freshenv).
     98     sphinx_app = Sphinx(srcdir, confdir, outdir, doctreedir, format,
---> 99                         confoverrides, None, None, True)
    100     sphinx_app.build(None, [rst_name])
    101     sys.path = old_sys_path

/home/vincent/miniconda3/envs/sage/lib/python3.6/site-packages/sphinx/application.py in __init__(self, srcdir, confdir, outdir, doctreedir, buildername, confoverrides, status, warning, freshenv, warningiserror, tags, verbosity, parallel, keep_going)
    151             if not path.isfile(path.join(self.confdir, 'conf.py')):
    152                 raise ApplicationError(__("config directory doesn't contain a "
--> 153                                           "conf.py file (%s)") % confdir)
    154 
    155         if not path.isdir(self.srcdir):

ApplicationError: config directory doesn't contain a conf.py file (/home/vincent/miniconda3/envs/sage/src/doc/en/introspect)
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/vincent/miniconda3/envs/sage:
#
# Name                    Version                   Build  Channel
_r-mutex                  1.0.0               anacondar_1  
alabaster                 0.7.12                     py_0    conda-forge
appdirs                   1.4.3                      py_1    conda-forge
arb                       2.16.0            hb20f59a_1000    conda-forge
asn1crypto                0.24.0                py36_1003    conda-forge
attrs                     19.1.0                     py_0    conda-forge
automat                   0.7.0                      py_1    conda-forge
babel                     2.7.0                      py_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.os              0.1.1                 py36_1001    conda-forge
bdw-gc                    8.0.4                h6bb024c_0    conda-forge
binutils_impl_linux-64    2.31.1               h6176602_1  
binutils_linux-64         2.31.1               h6176602_7  
bleach                    3.1.0                      py_0    conda-forge
boost-cpp                 1.69.0            h11c811c_1000    conda-forge
brial                     1.2.4                      py_1    conda-forge
bwidget                   1.9.11                        1  
bzip2                     1.0.6             h14c3975_1002    conda-forge
ca-certificates           2019.6.16            hecc5488_0    conda-forge
cairo                     1.16.0            h18b612c_1001    conda-forge
cddlib                    1!0.94j           ha14ba45_1001    conda-forge
certifi                   2019.6.16                py36_0    conda-forge
cffi                      1.12.3           py36h8022711_0    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
click                     7.0                        py_0    conda-forge
cliquer                   1.21                          0    conda-forge
constantly                15.1.0                     py_0    conda-forge
cryptography              2.7              py36h72c5cf5_0    conda-forge
curl                      7.64.1               hf8cf82a_0    conda-forge
cvxopt                    1.2.3           py36h915cfec_202    conda-forge
cycler                    0.10.0                     py_1    conda-forge
cypari2                   2.1.1            py36hef3d0d6_0    conda-forge
cysignals                 1.10.2           py36he8bbba0_1    conda-forge
cython                    0.29.10          py36he1b5a44_0    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
decorator                 4.4.0                      py_0    conda-forge
defusedxml                0.5.0                      py_1    conda-forge
docutils                  0.14                  py36_1001    conda-forge
dsdp                      5.8               h23d7751_1203    conda-forge
ecl                       16.1.2            h63faa73_1003    conda-forge
eclib                     20190226             h961da46_2    conda-forge
ecm                       7.0.4             ha14ba45_1002    conda-forge
entrypoints               0.3                   py36_1000    conda-forge
expat                     2.2.5             he1b5a44_1003    conda-forge
fastcache                 1.1.0            py36h516909a_0    conda-forge
fflas-ffpack              2.4.2                h042891f_1    conda-forge
fftw                      3.3.8           nompi_h7f3a6c3_1105    conda-forge
flask                     0.11.1                     py_1    conda-forge
flask-autoindex           0.6.2                      py_0    conda-forge
flask-babel               0.9                        py_0    conda-forge
flask-oldsessions         0.10                       py_0    conda-forge
flask-openid              1.2.5                 py36_1003    conda-forge
flask-silk                0.2                        py_0    conda-forge
flintqs                   1.0                           0    conda-forge
fontconfig                2.13.1            he4413a7_1000    conda-forge
fplll                     5.2.1             h4d64954_1000    conda-forge
fpylll                    0.4.1dev        py36h940fe96_1000    conda-forge
freetype                  2.10.0               he983fc9_0    conda-forge
future                    0.17.1                py36_1000    conda-forge
gap                       4.10.1               hee7d332_4    conda-forge
gap-core                  4.10.1               h711559f_4    conda-forge
gcc_impl_linux-64         7.3.0                habb00fd_1    conda-forge
gcc_linux-64              7.3.0                h553295d_7    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
gf2x                      1.2               ha03fbb7_1003    conda-forge
gfan                      0.6.2                h0eace8b_1    conda-forge
gfortran_impl_linux-64    7.3.0                hdf63c60_1  
gfortran_linux-64         7.3.0                h553295d_7  
giac                      1.5.0.37             h8b4616e_1    conda-forge
giflib                    5.1.9                h516909a_0    conda-forge
givaro                    4.1.1                h192cbe9_0    conda-forge
glib                      2.58.3            h6f030ca_1001    conda-forge
glpk                      4.65              ha14ba45_1001    conda-forge
gmp                       6.1.2             hf484d3e_1000    conda-forge
gmpy2                     2.1.0b1          py36h04dde30_0    conda-forge
graphite2                 1.3.13            hf484d3e_1000    conda-forge
gsl                       2.5                  h294904e_0    conda-forge
gst-plugins-base          1.14.5               h0935bb2_0    conda-forge
gstreamer                 1.14.5               h36ae1b5_0    conda-forge
gxx_impl_linux-64         7.3.0                hdf63c60_1    conda-forge
gxx_linux-64              7.3.0                h553295d_7    conda-forge
harfbuzz                  2.4.0                h37c48d4_1    conda-forge
hyperlink                 17.3.1                     py_0    conda-forge
icu                       58.2              hf484d3e_1000    conda-forge
idna                      2.8                   py36_1000    conda-forge
imagesize                 1.1.0                      py_0    conda-forge
iml                       1.0.5             h3057dec_1002    conda-forge
importlib_metadata        0.18                     py36_0    conda-forge
incremental               17.5.0                     py_0    conda-forge
ipykernel                 4.10.0                   py36_1    conda-forge
ipython                   5.8.0                    py36_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.4.2                      py_0    conda-forge
itsdangerous              1.1.0                      py_0    conda-forge
jinja2                    2.10.1                     py_0    conda-forge
jmol                      14.29.15                      1    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
jsonschema                3.0.0a3               py36_1000    conda-forge
jupyter_client            5.2.4                      py_3    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
kiwisolver                1.1.0            py36hc9558a2_0    conda-forge
krb5                      1.16.3            h05b26f9_1001    conda-forge
lcalc                     1.23              hab5a3d0_1002    conda-forge
libblas                   3.8.0                7_openblas    conda-forge
libbraiding               1.0                  he1b5a44_0    conda-forge
libbrial                  1.2.4                hcdf1f5c_0    conda-forge
libcblas                  3.8.0                7_openblas    conda-forge
libcurl                   7.64.1               hda55be3_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             he1b5a44_1006    conda-forge
libflint                  2.5.2             h3fb3072_1003    conda-forge
libgcc-ng                 9.1.0                hdf63c60_0  
libgd                     2.2.5             h0d07dcb_1005    conda-forge
libgfortran-ng            7.3.0                hdf63c60_0  
libhomfly                 1.02r6               h516909a_0    conda-forge
libiconv                  1.15              h516909a_1005    conda-forge
liblapack                 3.8.0                7_openblas    conda-forge
liblapacke                3.8.0                7_openblas    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libsemigroups             0.6.7                he1b5a44_0    conda-forge
libsodium                 1.0.16            h14c3975_1001    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.0.10            h57b8799_1003    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libwebp                   1.0.2                h576950b_1    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.9                h13577e0_0    conda-forge
linbox                    1.6.3                h03cd913_2    conda-forge
lrcalc                    1.2               hf484d3e_1000    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
m4ri                      20140914          he983fc9_1004    conda-forge
m4rie                     20150908          h3cfcefd_1000    conda-forge
make                      4.2.1             h14c3975_2004    conda-forge
markupsafe                1.1.1            py36h14c3975_0    conda-forge
matplotlib                2.2.3            py36hb69df0a_0  
maxima                    5.42.2               h63faa73_1    conda-forge
metis                     5.1.0             hf484d3e_1003    conda-forge
mistune                   0.8.4           py36h14c3975_1000    conda-forge
mpc                       1.1.0             hb20f59a_1006    conda-forge
mpfi                      1.5.4             hb20f59a_1001    conda-forge
mpfr                      4.0.2                ha14ba45_0    conda-forge
mpmath                    1.1.0                      py_0    conda-forge
nauty                     2.6.7             h14c3975_1000    conda-forge
nbconvert                 5.5.0                      py_0    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
networkx                  2.3                        py_0    conda-forge
normaliz                  3.7.3                h8b68381_1    conda-forge
notebook                  5.7.8                    py36_1    conda-forge
ntl                       11.3.2            h52520da_1000    conda-forge
numpy                     1.16.4           py36h95a1406_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openblas                  0.3.5             h9ac9557_1001    conda-forge
openjdk                   11.0.1            h516909a_1016    conda-forge
openssl                   1.1.1b               h14c3975_1    conda-forge
packaging                 19.0                       py_0    conda-forge
palp                      2.1                           0    conda-forge
pandoc                    2.7.3                         0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
pango                     1.40.14           h4ea9474_1004    conda-forge
pari                      2.11.1          pl526h7ba8bfb_0    conda-forge
path.py                   12.0.1                     py_0    conda-forge
pathlib2                  2.3.3                 py36_1000    conda-forge
pcre                      8.41              hf484d3e_1003    conda-forge
perl                      5.26.2            h516909a_1006    conda-forge
pexpect                   4.7.0                    py36_0    conda-forge
pickleshare               0.7.5                 py36_1000    conda-forge
pillow                    5.4.1           py36h00a061d_1000    conda-forge
pip                       19.1.1                   py36_0    conda-forge
pixman                    0.38.0            h516909a_1003    conda-forge
pkgconfig                 1.3.1                 py36_1001    conda-forge
planarity                 3.0.0.5           h14c3975_1002    conda-forge
ppl                       1.2               h64e95d0_1002    conda-forge
pplpy                     0.8.4            py36h9603a25_0    conda-forge
prometheus_client         0.7.1                      py_0    conda-forge
prompt_toolkit            1.0.15                     py_1    conda-forge
psutil                    5.6.3            py36h516909a_0    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pyasn1                    0.4.5                      py_0    conda-forge
pyasn1-modules            0.2.5                      py_0    conda-forge
pycparser                 2.19                     py36_1    conda-forge
pygments                  2.4.2                      py_0    conda-forge
pyhamcrest                1.9.0                      py_2    conda-forge
pynac                     0.7.25           py36h58d1e65_3    conda-forge
pyopenssl                 19.0.0                   py36_0    conda-forge
pyparsing                 2.4.0                      py_0    conda-forge
pyqt                      5.9.2            py36hcca6a23_0    conda-forge
pyrsistent                0.15.2           py36h516909a_0    conda-forge
pysocks                   1.7.0                    py36_0    conda-forge
python                    3.6.7             h381d211_1004    conda-forge
python-dateutil           2.8.0                      py_0    conda-forge
python3-openid            3.1.0                 py36_1001    conda-forge
pytz                      2019.1                     py_0    conda-forge
pyzmq                     17.1.2           py36he6710b0_2  
qd                        2.3.22            h1c2f66e_1002    conda-forge
qt                        5.9.7                h52cfd70_2    conda-forge
r-assertthat              0.2.1            r351h6115d3f_0    conda-forge
r-backports               1.1.4            r351hcdcec82_0    conda-forge
r-base                    3.5.1             hea2b9be_1007    conda-forge
r-bh                      1.69.0_1         r351h6115d3f_0    conda-forge
r-bit                     1.1_14           r351hcdcec82_0    conda-forge
r-bit64                   0.9_7           r351h96ca727_1000    conda-forge
r-blob                    1.1.1                 r351_1001    conda-forge
r-cli                     1.1.0            r351h6115d3f_0    conda-forge
r-crayon                  1.3.4           r351h6115d3f_1001    conda-forge
r-dbi                     1.0.0           r351h6115d3f_1001    conda-forge
r-dbplyr                  1.4.2             r35h6115d3f_0    conda-forge
r-digest                  0.6.19           r351h0357c0b_0    conda-forge
r-dplyr                   0.8.1            r351h0357c0b_0    conda-forge
r-fansi                   0.4.0           r351h96ca727_1000    conda-forge
r-glue                    1.3.1            r351hcdcec82_0    conda-forge
r-magrittr                1.5             r351h6115d3f_1001    conda-forge
r-memoise                 1.1.0           r351h6115d3f_1001    conda-forge
r-pillar                  1.4.1                h6115d3f_0    conda-forge
r-pkgconfig               2.0.2           r351h6115d3f_1001    conda-forge
r-plogr                   0.2.0           r351h6115d3f_1001    conda-forge
r-prettyunits             1.0.2           r351h6115d3f_1001    conda-forge
r-purrr                   0.3.2            r351hcdcec82_0    conda-forge
r-r6                      2.4.0            r351h6115d3f_0    conda-forge
r-rcpp                    1.0.1            r351h0357c0b_0    conda-forge
r-rlang                   0.3.4            r351hcdcec82_0    conda-forge
r-rsqlite                 2.1.1           r351h29659fb_1000    conda-forge
r-tibble                  2.1.3             r35hcdcec82_0    conda-forge
r-tidyselect              0.2.5           r351h29659fb_1000    conda-forge
r-utf8                    1.1.4           r351h96ca727_1000    conda-forge
r-vctrs                   0.1.0           r351h96ca727_1000    conda-forge
r-zeallot                 0.1.0           r351h6115d3f_1000    conda-forge
ratpoints                 2.1.3             ha14ba45_1002    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
requests                  2.22.0                   py36_0    conda-forge
rpy2                      2.9.4           py36r351h941a26a_1    conda-forge
rubiks                    20070912          hf484d3e_1001    conda-forge
rw                        0.7                           1    conda-forge
sage                      8.8.rc1          py36hcb32578_0    conda-forge
sagelib                   8.8.rc1          py36h21c7cb8_1    conda-forge
sagemath-db-combinatorial-designs 20140630                      1    conda-forge
sagemath-db-conway-polynomials 0.5.0            py36hecc5488_0    conda-forge
sagemath-db-elliptic-curves 0.8.1                hecc5488_0    conda-forge
sagemath-db-graphs        20161026                      1    conda-forge
sagemath-db-polytopes     20170220                      1    conda-forge
sagenb                    1.1.2                      py_0    conda-forge
scipy                     1.3.0            py36h921218d_0    conda-forge
send2trash                1.5.0                      py_0    conda-forge
service_identity          18.1.0                     py_0    conda-forge
setuptools                41.0.1                   py36_0    conda-forge
simplegeneric             0.8.1                      py_1    conda-forge
singledispatch            3.4.0.3               py36_1000    conda-forge
singular                  4.1.2.p1             hdd094a0_2    conda-forge
sip                       4.19.8          py36hf484d3e_1000    conda-forge
six                       1.12.0                py36_1000    conda-forge
snowballstemmer           1.2.1                      py_1    conda-forge
speaklater                1.3                        py_3    conda-forge
sphinx                    1.8.5                    py36_0    conda-forge
sphinxcontrib-websupport  1.1.2                      py_0    conda-forge
sqlite                    3.28.0               h8b20d00_0    conda-forge
suitesparse               4.5.6             heab0a99_1202    conda-forge
symmetrica                2.0                           0    conda-forge
sympow                    1.018.1                       1    conda-forge
sympy                     1.4                      py36_0    conda-forge
tachyon                   0.99b6                        0    conda-forge
tbb                       2019.7               hc9558a2_0    conda-forge
terminado                 0.8.2                    py36_0    conda-forge
testpath                  0.4.2                   py_1001    conda-forge
tk                        8.6.9             hed695b0_1002    conda-forge
tktable                   2.10                 h14c3975_0  
tornado                   4.5.3           py36h14c3975_1001    conda-forge
traitlets                 4.3.2                 py36_1000    conda-forge
twisted                   18.9.0          py36h14c3975_1000    conda-forge
tzlocal                   1.5.1                      py_0    conda-forge
urllib3                   1.24.3                   py36_0    conda-forge
wcwidth                   0.1.7                      py_1    conda-forge
webencodings              0.5.1                      py_1    conda-forge
werkzeug                  0.15.4                     py_0    conda-forge
wheel                     0.33.4                   py36_0    conda-forge
widgetsnbextension        3.4.2                 py36_1000    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.9             h516909a_1004    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.7             h14c3975_1000    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
zeromq                    4.3.1             hf484d3e_1000    conda-forge
zipp                      0.5.1                      py_0    conda-forge
zlib                      1.2.11            h14c3975_1004    conda-forge
zn_poly                   0.9.1                he80fd80_0    conda-forge
zope.interface            4.6.0           py36h14c3975_1000    conda-forge
zstd                      1.4.0                h3b9ef0a_0    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : sage
    active env location : /home/vincent/miniconda3/envs/sage
            shell level : 1
       user config file : /home/vincent/.condarc
 populated config files : /home/vincent/.condarc
          conda version : 4.6.14
    conda-build version : not installed
         python version : 3.7.3.final.0
       base environment : /home/vincent/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/vincent/miniconda3/pkgs
                          /home/vincent/.conda/pkgs
       envs directories : /home/vincent/miniconda3/envs
                          /home/vincent/.conda/envs
               platform : linux-64
             user-agent : conda/4.6.14 requests/2.22.0 CPython/3.7.3 Linux/5.1.6-arch1-1-ARCH arch/rolling glibc/2.29
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```
</details>

**Comment by isuruf** [comment]
@saraedum, any progress on https://trac.sagemath.org/ticket/25786?

**Comment by saraedum** [comment]
My changes there don't change anything™ but some doctests fail. I'd have to talk to somebody who understands this part of the code better…

**Comment by isuruf** [comment]
This is fixed in 8.8. I used changes from 25786 and also installed some doc sources in `PREFIX/share/sage/docsrc` and fixed `SAGE_DOC_SRC` variable
